api_test.rb

```diff
@@ -27,11 +27,14 @@
 
 GlobusClient.disallow_writes(user_id:, path:)
 
+files_list = GlobusClient.get_filenames(user_id:, path: "#{GlobusClient.config.uploads_directory}/#{path}")
```

Comment:
Did you find that we need the explicit uploads directory at the front of the path here? I was thinking https://github.com/sul-dlss/globus_client/pull/50/files#diff-8db5ac815478941488dcb86e4727dbc7de1a80b352d94b258fdb18c241024a1eR86 would take care of that, like it should be doing for the other path-related methods in `Endpoint`.

Reply:
The link is taking me to a spec--can you help me see what I should take away from that? I do understand the part about trying to get this to more like the other `Endpoint` methods and am trying to align this better with that.

Reply:
@lwrubel Doh! Sorry, this is the method I was referring to: https://github.com/sul-dlss/globus_client/blob/main/lib/globus_client/endpoint.rb#L75-L85

Specifically line 82 which joins the base directory to the user-provided path.

Reply:
Got it.

Reply:
Added the base directory to the user path in the `get_filenames` method.